CHANGES

```diff
@@ -9,6 +9,9 @@
 
  Changes between 1.1.1h and 1.1.1i [xx XXX xxxx]
 
+  *) Add support for Apple Sillicon M1 Macs with the darwin64-arm64-cc target.
```

Comment:
```suggestion
  *) Add support for Apple Silicon M1 Macs with the darwin64-arm64-cc target.
```

Reply:
Thanks, I have the same typo in the commit message, will fix ASAP.